src/main/java/com/github/jtendermint/jabci/CodeType.java

```diff
@@ -0,0 +1,372 @@
+package com.github.jtendermint.jabci;
+
+public enum CodeType
```

Comment:
Another idea could be to just use a class with constants instead of enum. So we can get rid of all the protobuf things here. In the end, abci/tendermint just expects an INT as code.

Reply:
You are right. We can declare a class, rather than using the enum generated by protobuf, which is removed from types.proto now.